### ISSUE TITLE
Taxa labels dont show up after performing plot_diff_cladogram

### ISSUE THREAD

**Issue by BiggusDickus666** [ChiLiubio/microeco#367]
Hi,
I have recently performed the representation of differential abundance analysis as a cladogram following Microeco guide. I noticed that taxa names do not appear in the cladogram. What may be causing the problem? 
Here you can find attached the cladogram

Thank you very much in advance!!

[diffMicroecoTripIno.pdf](https://github.com/ChiLiubio/microeco/files/15404493/diffMicroecoTripIno.pdf)

**Comment by ChiLiubio** [comment]
Hi. Please attach your microtable data and steps so that I can reproduce your issue? To save the dataset, please follow the steps in the tutorial (https://chiliubio.github.io/microeco_tutorial/notes.html#save-function) and attach the compressed object. I will have a try.

**Comment by BiggusDickus666** [comment]
Hi @ChiLiubio ,
Here you can find attached my microtable object.
Thank you very much for your help!!

[Uploading MicroecoIno.zip…]()

**Comment by ChiLiubio** [comment]
Hi. The link can not be opened. Please make sure it is available. Please also provide the R script.

**Comment by BiggusDickus666** [comment]
Hi @ChiLiubio 
Sorry for that, hope this link work!
Thanks for your patience
[MicroecoIno.zip](https://github.com/ChiLiubio/microeco/files/15429690/MicroecoIno.zip)

**Comment by BiggusDickus666** [comment]
Hi @ChiLiubio 
I contrasted my info session with yours along with the info session of a lab partner of mine, and I noticed it was all about the R version. Mine was R4.2. We run the same code utilizing the same .qza in a newer R version, and labels appeared on the cladogram without a problem. Something as simple as that.
Thank you very much for your time and effort, and sorry for any inconvenience!!
